src/pages/Activity/components/ExercisePopupBox.jsx

```diff
@@ -5,25 +5,37 @@ import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
 import Button from '@mui/material/Button';
+import { Typography, Box } from '@mui/material';
 import Calendar from './Calendar';
 import SportsDropDown from './SportsDropDown';
+import StartEndTime from './StartEndTime';
+import Duration from './Duration';
+import styles from '../Activity.module.css';
 
 const ExercisePopupBox = React.forwardRef(({ title, message, onConfirm, isDelete }, ref) => {
+
+  // Sample data for the sports dropdown
+  const sports = ['Soccer', 'Basketball', 'Tennis', 'Baseball', 'Cricket'];
```

Comment:
```suggestion
  const sports = ['Football', 'Basketball', 'Tennis', 'Baseball', 'Cricket'];
```

Sorry, Jen, we need to call it _Football_... Otherwise, everything looks great.

Reply:
No worries! I committed it, but I think it needs 2 approvals again